cli/smartcontract/generate.go

```diff
@@ -30,17 +30,22 @@ var generatorFlags = []cli.Flag{
 	},
 	cli.StringFlag{
 		Name:  "hash",
-		Usage: "Smart-contract hash",
+		Usage: "Smart-contract hash. If not passed, the wrapper will be designed for dynamic hash usage",
 	},
 }
 
 var generateWrapperCmd = cli.Command{
-	Name:        "generate-wrapper",
-	Usage:       "generate wrapper to use in other contracts",
-	UsageText:   "neo-go contract generate-wrapper --manifest <file.json> --out <file.go> --hash <hash> [--config <config>]",
-	Description: ``,
-	Action:      contractGenerateWrapper,
-	Flags:       generatorFlags,
+	Name:      "generate-wrapper",
+	Usage:     "generate wrapper to use in other contracts",
+	UsageText: "neo-go contract generate-wrapper --manifest <file.json> --out <file.go> [--hash <hash>] [--config <config>]",
+	Description: `Generates a Go wrapper for interacting with a smart contract. If the
```

Comment:
s/`Generates a Go wrapper for interacting with a smart contract.`/`Generates a Go wrapper to use it in other smart contracts.`